diff -aurN galago-sharp-0.3.2.orig/configure.ac galago-sharp-0.3.2/configure.ac
--- galago-sharp-0.3.2.orig/configure.ac	2005-06-08 08:05:58.000000000 +0200
+++ galago-sharp-0.3.2/configure.ac	2006-03-07 18:23:12.943282000 +0100
@@ -75,7 +75,7 @@
 	AC_PATH_PROG(RUNTIME, mono, no)
 	AC_PATH_PROG(CSC, mcs, no)
 
-	GACUTIL_FLAGS='/package $(PACKAGE) /gacdir $(libdir) /root $(DESTDIR)$(libdir)'
+	GACUTIL_FLAGS='/package $(PACKAGE) /gacdir $(libdir) /root $(DESTDIR)$(prefix)/lib'
 
 	if test `uname -s` = "Darwin"; then
 		LIB_PREFIX=
