diff -aurN galago-sharp-0.3.2.orig/galago-sharp.pc.in galago-sharp-0.3.2/galago-sharp.pc.in
--- galago-sharp-0.3.2.orig/galago-sharp.pc.in	2005-02-10 19:20:31.000000000 +0100
+++ galago-sharp-0.3.2/galago-sharp.pc.in	2006-03-07 18:16:01.790827136 +0100
@@ -1,6 +1,6 @@
 prefix=@prefix@
 exec_prefix=@exec_prefix@
-libdir=@libdir@
+libdir=${exec_prefix}/lib
 
 Name: Galago#
 Description: Galago# - Galago .NET Bindings
